utils.groovy

```diff
@@ -123,8 +123,22 @@ def get_source_config_ref_for_stream(pipecfg, stream) {
     }
 }
 
-def get_use_osbuild_for_stream(pipecfg, stream) {
-    return pipecfg.streams[stream].use_osbuild ? '1' : ''
+def get_env_vars_for_stream(pipecfg, stream) {
+    env = [:]
+    // Grab the globally configured env vars first
+    if (pipecfg.env) {
+        for (x in pipecfg.env) {
+            env[x.key] = x.value
+        }
```

Comment:
Isn't that just

```groovy
env.putAll(pipecfg.env)
```

?

Similarly below.

Reply:
maybe :) - I use groovy about once every 3-6 months so I always end up dropping back to languange non-specific primitives like a for loop.

I'll have to test to make sure `putAll` gives us what we want here.